homeassistant/components/input_boolean.py

```diff
@@ -61,15 +62,16 @@ def setup(hass, config):
 
     entities = []
 
-    for object_id, cfg in config[DOMAIN].items():
-        if not cfg:
-            cfg = {}
+    for _, p_config in config_per_platform(config, DOMAIN):
```

Comment:
Instead of `config_per_platform`, use `extract_domain_configs` (also in helpers package)